tests/integration/test_database_delta/test.py

```diff
@@ -101,7 +101,7 @@ def test_embedded_database_and_tables(started_cluster, use_delta_kernel):
     node1.query(f"drop database if exists unity_test_{test_uuid}")
     node1.query(
         f"create database unity_test_{test_uuid} engine DataLakeCatalog('http://localhost:8080/api/2.1/unity-catalog') settings warehouse = 'unity', catalog_type='unity', vended_credentials=false, allow_experimental_delta_kernel_rs={use_delta_kernel}",
-        settings={"allow_experimental_database_unity_catalog": "1"},
+        settings={"allow_database_unity_catalog": "1"},
```

Comment:
Let's leave the old setting `allow_experimental_database_unity_catalog` in some of the tests to make sure it still works along with the new one.

Reply:
I also changed it for glue and Iceberg rest